packages/one-bx-shell-app/src/assets/config/plugins-schema.json

```diff
@@ -0,0 +1,159 @@
+{
+  "id": "sap-plugins-schema",
+  "version": "0.0.1",
```

Comment:
Can schem have also a description for some significant fields ?

Reply:
Yes

Reply:
@skocharian can you create new tasks, to comment fields that are not obvios from its name